lib/Index/StoreSymbolRecord.cpp

```diff
@@ -289,6 +289,20 @@ bool StoreSymbolRecord::foreachCoreSymbolData(function_ref<bool(StringRef USR,
   return !Err && Finished;
 }
 
+bool StoreSymbolRecord::foreachSymbolOccurrence(function_ref<bool(SymbolOccurrenceRef Occur)> Receiver) {
+  bool Finished;
+  bool Err = doForData([&](IndexRecordReader &Reader) {
+      // Return all occurrences.
```

Comment:
Indentation of the comment seems to be off

Reply:
done